anomaly_detector/storage/storage_catalog.py

```diff
@@ -53,12 +54,19 @@ def _kafka_datasink_api(cls, config):
         logging.info("save kafka datasink")
         return KafkaSink(config=config)
 
+    @classmethod
+    def _stdout_datasink_api(cls, config):
+        """Kafka data sink."""
```

Comment:
I think you forgot to edit this after copy pasting ;p

Reply:
Fixed